### PR TITLE
#1033 CDXML: expanded abbreviation opens as contracted from .cdxm files

### PR DESCRIPTION
**Pull request by dmalysh**

### CODE REVIEW

File: api/tests/integration/tests/formats/cdxml_ket_expanded.py
Comment (AliaksandrDziarkach) [inline api/tests/integration/tests/formats/cdxml_ket_expanded.py:1]: I think you have to add corresponding out file api/tests/integration/ref/formats/cdxml_ket_expanded.py.out

Comment (dmalysh) [reply api/tests/integration/tests/formats/cdxml_ket_expanded.py:1]: I added contracted_expanded_structures.ket file as an expected result of test purposes. Is it okay?

Comment (AliaksandrDziarkach) [reply api/tests/integration/tests/formats/cdxml_ket_expanded.py:1]: No you should provide reference output file.
just do "python api/tests/integration/tests/formats/cdxml_ket_expanded.py >  api/tests/integration/ref/formats/cdxml_ket_expanded.py.out"
You can check result of integration test by
python api/tests/integration/test.py -t 1 -p cdxml_ket_expanded.py
Now it return "formats/cdxml_ket_expanded.py........................................[NEW]    0.01 sec" (see "Test" at https://github.com/epam/Indigo/actions/runs/5749220886/job/15584133387?pr=1209)
But should return "formats/cdxml_ket_expanded.py........................................[PASSED]    0.01 sec

Comment (dmalysh) [reply api/tests/integration/tests/formats/cdxml_ket_expanded.py:1]: Done.

**Review by AliaksandrDziarkach — APPROVED** [review]
LGTM

**Pull request merged**